### PR TITLE
Activity Log - added discarded styles and fixed up other bugs

### PR DESCRIPTION
**Pull request by MichaelArestad**
This is a bit of a doozy for style changes. I tackled a bunch while I was in there since I had to shuffle some things around anyway.

#### Notes
* removed unused styles
* refactored for simpler styling
* removed unused div
* added classes to be more explicit with selectors
* added discarded styles: Fixes https://github.com/Automattic/wp-calypso/issues/19132 https://github.com/Automattic/wp-calypso/issues/18759
* rebuilt timelines to be friendlier to work with (and use fewer elements)
* converted em to span in the rewind button
* aligned items to their icons better
* made text colors a bit more accessible
* made sure text sizes matched our standards
* changed color of Jetpack icon: fixes https://github.com/Automattic/wp-calypso/issues/18862
* Removed overflowing line: Fixes https://github.com/Automattic/wp-calypso/issues/17065

#### Before
image
image

#### After
image
image

#### What I wasn't able to do:
( @elio or @dmsnell maybe one of you could do one of these in a followup PR? )

1. I would like to show the cards above the rewind confirmation dialog as `is-discarded` which will detach the rewind dialog from them at the same time.  I don't think any CSS changes would be needed.

2. I would like to add an `is-discarded` class to `activity-log-day` whenever every event in the day is detached.

3. On Posts, Pages, and Comments, I would like to make the name of the post, page, or comment a link that the user can follow to the post in their dashboard (unless discarded, of course).

### CODE REVIEW

**Comment by dmsnell** [comment]
> I would like to show the cards above the rewind confirmation dialog as is-discarded which will detach the rewind dialog from them at the same time. I don't think any CSS changes would be needed.

This is something on my radar too. If we simply wait just a bit then we can get this for free. Right now we assume that the perspective from which `isDiscarded` is calculated is always _now_, but as soon as we pull that up a couple levels we can recalculate it on the fly which will automatically get it there when the rewind dialog is open.

**Comment by rickybanister** [comment]
Beautiful stuff. The black and white avatar is a nice touch :)

Glad to see the funny errant line gone too. 

Ship it.

File: client/my-sites/stats/activity-log-day/index.jsx [hunk 1]
```diff
@@ -157,8 +157,8 @@ class ActivityLogDay extends Component {
 				primary={ 'primary' === type }
 			>
 				<Gridicon icon="history" size={ 18 } />{' '}
-				{ this.props.translate( 'Rewind {{em}}to this day{{/em}}', {
-					components: { em: <em /> },
+				{ this.props.translate( 'Rewind {{span}}to this day{{/span}}', {
```
Comment (a8ci18n) [inline client/my-sites/stats/activity-log-day/index.jsx]: :disappointed: 17 existing translations will be lost with this change.

**Comment by MichaelArestad** [comment]
@dmsnell @rickybanister thanks for taking a look!

**Pull request merged**